### PR TITLE
Change interactive add cred to use interact toolkit

### PR DESCRIPTION
**Pull request by natefinch**

### CODE REVIEW

File: cmd/juju/cloud/addcredential.go [hunk 1]
```diff
@@ -392,47 +309,50 @@ func (c *addCredentialCommand) promptCredentialAttributes(
...
+		return p.Enter(name)
 	}
-	value := strings.TrimSpace(input)
-	return value, nil
 }
 
-func (c *addCredentialCommand) readHiddenField(in io.Reader) (string, error) {
-	if f, ok := in.(*os.File); ok && terminal.IsTerminal(int(f.Fd())) {
-		value, err := terminal.ReadPassword(int(f.Fd()))
+func enterFile(name string, p *interact.Pollster) (string, error) {
```
Comment (reedobrien) [inline cmd/juju/cloud/addcredential.go]: Seems like prompting for a file might be a common enough case to add to pollster. Maybe.

**Comment by reedobrien** [comment]
!!build!!

**Comment by reedobrien** [comment]
LGTM once conflicts are resolved and prelim tests pass. 

Also, I don't see any QA steps.

**Comment by natefinch** [comment]
!!build!!

**Comment by natefinch** [comment]
!!build!!

**Comment by natefinch** [comment]
imma try to land this as-is
$$merge$$

**Pull request merged**